pages/genre/genre.go

```diff
@@ -42,3 +45,22 @@ func containsLowerCase(array []string, search string) bool {
 
 	return false
 }
+
+func averageGenreScore(user *arn.User, animes []*arn.Anime) string {
+	if user == nil {
+		return ""
+	}
+
+	counter := 0.0
+	scores := 0.0
+
+	for _, anime := range animes {
+
+		if user.AnimeList().Contains(anime.ID) {
+			scores = scores + user.AnimeList().Find(anime.ID).Rating.Overall
+			counter = counter + 1
+		}
+	}
+
+	return strconv.FormatFloat(scores/counter, 'f', 6, 64)
```

Comment:
1.) Rather than formatting it here, I'd prefer to return the float and format it in the template file (pixy).
2.) This doesn't handle the case of `counter == 0`, it would divide by zero if no matching items are found.